src/main.ts

```diff
@@ -133,7 +135,21 @@ for await (const app of apps) {
         appRoles: app.appRoles,
       });
     }
+
+    if (
+      app.preferredSingleSignOnMode &&
+      app.preferredSingleSignOnMode == "saml"
+    ) {
+      await addSsoSamlUris({
+        token: token,
+        appId: applicationId,
+        identifierUrls: app.identifierUrls,
+        redirectUrls: app.redirectUrls,
+      });
+    }
```

Comment:
why do we need this? it's already here?
https://github.com/hmcts/azure-app-proxy-manager/blob/59649c213b24b7441827245b6d1a79214209343d/src/applicationManager.ts#L236-L238

Reply:
The `updateApplicationConfig` function doesnt appear to deploy the identifier Urls first time, I had to re-run for those and the extra redirect Urls to be added, very possible I'm missing something though.